modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/contents/ProductCardPersona.tsx

```diff
@@ -51,10 +51,10 @@ const ProductCardPersona = () => {
 										)}
```

Comment:
Please change the name of this file to ```PersonalProductCard.text``` and change this information where it is being used

Reply:
Done !